_config.yml

```diff
@@ -679,6 +679,14 @@ gitalk:
   # Available values: en, es-ES, fr, ru, zh-CN, zh-TW
   language:
 
+# utteranc
+# Demo: https://utteranc.es/  http://trumandu.github.io/about/
+# theme: github-light,github-dark,github-dark-orange
```

Comment:
You missed three themes, common user maybe won't find it.
https://github.com/utterance/utterances/tree/master/src/stylesheets/themes